lib/status/index.js

```diff
@@ -0,0 +1,17 @@
+import angular from 'angular';
+
+var status = angular.module('guStatus', []);
+
+status.directive('guStatus', function statusDirective(){
+  return {
+    restrict: 'E',
+    replace: true,
+    transclude: true,
+    template: (el, attrs) => {
```

Comment:
Agreed. For the use case you're using here, its better to set a constant value here rather than in the link function as the link function operates on generated dom nodes.